src/ert/_c_wrappers/enkf/rng_config.py

```diff
@@ -84,4 +84,6 @@ def format_seed(random_seed: str):
             fseed[i] += int(random_seed[seed_pos])
             seed_pos = (seed_pos + 1) % len(random_seed)
 
-    return b"".join(struct.pack("I", x) for x in fseed)
+    # The function this was derived from had integer overflow, so we
+    # allow for the same here
+    return b"".join(struct.pack("I", x % (2**32 - 1)) for x in fseed)
```

Comment:
Now I am a bit unsure. `x % x == 0` so should this be `x % (2**32)` ?

Reply:
You are right, fixed